### PR TITLE
Update create-sso-office-add-ins-aspnet.md

### PR DESCRIPTION
**Pull request by joelfmrodrigues**
removed reference for Node and express

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c4fe4ce](https://github.com/joelfmrodrigues/office-js-docs-pr/commits/c4fe4ce3666aa21e59a10430860c93ea3afd55dc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/joelfmrodrigues/office-js-docs-pr/blob/patch-1/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet?branch=pr-en-us-2026) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202007171021559858-2026%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=eQzlR5N3mkf7lQIfAScAs3qep4bGI8POOFoSjPZ6Ds0%3D&st=2020-07-17T10%3A18%3A48Z&se=2020-08-17T10%3A23%3A48Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/develop/create-sso-office-add-ins-aspnet.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ localization_priority: Normal
 # Create an ASP.NET Office Add-in that uses single sign-on (preview)
 
 When users are signed in to Office, your add-in can use the same credentials to permit users to access multiple applications without requiring them to sign in a second time. For an overview, see [Enable SSO in an Office Add-in](sso-in-office-add-ins.md).
-This article walks you through the process of enabling single sign-on (SSO) in an add-in that is built with Node.js and Express.
+This article walks you through the process of enabling single sign-on (SSO) in an add-in that is built with ASP.NET.
 
 > [!NOTE]
 > For a similar article about an ASP.NET-based add-in, see [Create a Node.js Office Add-in that uses single sign-on](create-sso-office-add-ins-nodejs.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/create-sso-office-add-ins-aspnet.md:14]:
```suggestion
> For a similar article about a Node.js-based add-in, see [Create a Node.js Office Add-in that uses single sign-on](create-sso-office-add-ins-nodejs.md).
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bbc8fd6](https://github.com/joelfmrodrigues/office-js-docs-pr/commits/bbc8fd6e1ce79a716fff3b27a5d9096622bba27e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/joelfmrodrigues/office-js-docs-pr/blob/patch-1/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet?branch=pr-en-us-2026) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202007171659062606-2026%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5yDwe8gDECkqteyALTSlPVhI8r1bhgdH%2F3cfBSGd6sI%3D&st=2020-07-17T16%3A55%3A46Z&se=2020-08-17T17%3A00%3A46Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by ElizabethSamuel-MSFT** [comment]
@joelfmrodrigues Thanks for making this updating!

**Pull request merged**